### PR TITLE
Added Farm Footer values

### PR DESCRIPTION
**Pull request by vkulinich-cl**
-Added claimable value and Button for claiming all farms rewards within Pool Footer
-Added a FF for formatting the claimable value, which is set to false so far.

### CODE REVIEW

File: src/i18n/locales/en/translations.json [hunk 1]
```diff
@@ -251,13 +252,18 @@
...
+  "farms.modal.details.joinedOn.label": "Joined on:",
+  "farms.modal.details.value.label": "Value:",
   "farms.modal.joinedFarms.joined.label": "Your farm positions",
   "farms.modal.joinedFarms.available.label": "Available farms",
   "farms.modal.joinedFarms.button.exit.label": "Exit All Farms",
   "farms.modal.exit.toast.onLoading": "<0>Withdrawing </0><1>{{ amount, bignumber(type: 'token') }}</1> <0>Shares from farm</0>",
   "farms.modal.exit.toast.onSuccess": "<0>You withdrawed </0><1>{{ amount, bignumber(type: 'token') }}</1> <0>Shares from farm</0>",
   "farms.modal.joinedFarms.button.joinAll.label": "Join All Free Farms",
   "farms.claimCard.title": "Claimable Now",
+  "farms.claimCard.smallValue": "Claimable  below 0.01$",
```
Comment (obsivac-cleevio) [inline src/i18n/locales/en/translations.json:266]:
```suggestion
  "farms.claimCard.smallValue": "Claimable below 0.01$",
```

File: src/sections/pools/farms/modals/details/FarmDetailsModal.tsx [hunk 2]
```diff
@@ -8,26 +8,101 @@ import { Spacer } from "components/Spacer/Spacer"
...
 }
 
+type FarmDetailsModalValuesProps = {
+  pool: OmnipoolPool
+  depositNft: DepositNftType
+  enteredBlock: BigNumber
+  yieldFarmId: string
+}
+
+const FarmDetailsModalValues = ({
```
Comment (obsivac-cleevio) [inline src/sections/pools/farms/modals/details/FarmDetailsModal.tsx:35]: tota nova komponenta muze jit klidne do svojeho souboru zvlast

File: src/sections/pools/farms/position/FarmingPosition.utils.tsx [hunk 3]
```diff
@@ -6,26 +6,30 @@ import { u32 } from "@polkadot/types"
...
+import { useOmnipoolAsset, useOmnipoolPositions } from "api/omnipool"
 import { useSpotPrice } from "api/spotPrice"
 import { OMNIPOOL_ACCOUNT_ADDRESS } from "utils/api"
 import { BN_10, BN_NAN } from "utils/constants"
 import BN from "bignumber.js"
 import { useQueryReduce } from "utils/helpers"
+import { isNotNil } from "utils/helpers"
 
-export const useDepositShare = (poolId: u32, depositNftId: string) => {
+export const useDepositShare = (poolId: u32, depositNftId: string[]) => {
```
Comment (obsivac-cleevio) [inline src/sections/pools/farms/position/FarmingPosition.utils.tsx:18]:
```suggestion
export const useDepositShare = (poolId: u32, depositNftIds: string[]) => {
```

File: src/sections/pools/pool/footer/PoolFooter.tsx [hunk 4]
```diff
@@ -1,28 +1,88 @@
...
+    const msType = type === "onError" ? "onLoading" : type
+    memo[type] = (
+      <>
+        <Trans i18nKey={`farms.claimCard.toast.${msType}`}>
+          <span />
+        </Trans>
+        {t("value", {
+          value: claimable.data?.usd,
+          type: "token",
+          suffixPrefix: "$",
```
Comment (obsivac-cleevio) [inline src/sections/pools/pool/footer/PoolFooter.tsx:32]: tady asi spis `numberPrefix`? resp proc ne jen `type: 'dollar'`?

**Review by obsivac-cleevio — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en/translations.json [hunk 5]
```diff
@@ -65,7 +65,8 @@
   "liquidity.asset.actions.myPositions": "My positions",
   "liquidity.asset.claim.button": "Claim all",
   "liquidity.asset.claim.total": "Total locked {{ locked, bignumber(numberPrefix: '$'; type: 'dollar') }}",
-  "liquidity.asset.claim.claimable": "{{ claimable, bignumber(numberPrefix: '$'; type: 'dollar') }} claimable",
+  "liquidity.asset.claim.farmTotal": "Total locked {{ locked, bignumber(numberPrefix: '$'; type: 'dollar') }} of which {{ available, bignumber(numberPrefix: '$'; type: 'dollar') }} not farming.",
```
[the remark this answers is not in the document]
Comment (vkulinich-cl) [reply src/i18n/locales/en/translations.json:68]: musi to tam byt, bez toho ten prefix se tam nezobrazi

**Review by obsivac-cleevio — APPROVED** [review]

**Pull request merged**